test/kokoro/xds_k8s.cfg

```diff
@@ -0,0 +1,13 @@
+# Config file for internal CI
+
+# Location of the continuous shell script in repository.
+build_file: "grpc-go/test/kokoro/xds-k8s.sh"
+timeout_mins: 120
```

Comment:
Is this enough?  We just bumped this for the non-security interop tests due to test case expansion.

Reply:
From my local runs, the security tests take around 40 minutes at this point in time.